### DISCUSSION TITLE
subscriptions-transport-ws is not maintained anymore

### DISCUSSION THREAD

**Discussion by diegoglezbahamonde** [the-guild-org/apollo-angular#1778] [General]

**Comment by KeithGillette** [comment]
Yes, I think think the Apollo Angular documentation should be updated to use the Apollo-recommended `graphql-ws` WebSockets implementation. We've replaced `subscriptions-transport-ws` with `graphql-ws` in our Apollo Angular project following the [`graphql-ws` Apollo Client Web Recipe](https://github.com/enisdenjo/graphql-ws#recipes) but are still figuring out how to cleanly unsubscribe from all GraphQL Subscriptions on user sign-out, as `graphql-ws` doesn't seem to have an equivalent to the `subscriptions-transport-ws` `SubscriptionClient.unsubscribeAll()`.
